src/main/java/uk/gov/hmcts/darts/common/entity/DailyListEntity.java

```diff
@@ -107,4 +105,10 @@ public class DailyListEntity extends CreatedModifiedBaseEntity {
     @JoinColumn(name = "elt_id")
     private ExternalLocationTypeEntity externalLocationTypeEntity;
 
+    @Column(name = "subcontent_object_id")
+    private String subcontentObjectId;
+
+    @Column(name = "subcontent_position")
+    private Integer subcontentPosition;
```

Comment:
Validation (See previous comment)?

Reply:
See earlier response to similar comment.